lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -210,6 +211,20 @@ def invalid_type_error(manifest)
         def missing_keys_validation_error(missing_keys)
           ValidationError.new('manifest_keys.missing', missing_keys: missing_keys.join(', '), count: missing_keys.length)
         end
+
+        def framework_version_iframe_only(manifest)
+          is_iframe_app = manifest['location'].any? do |host, locations|
```

Comment:
Can we somehow reuse the code [here](https://github.com/zendesk/zendesk_apps_support/blob/master/lib/zendesk_apps_support/package.rb#L223-L232)?